distribution/src/main/release/samples/jax_rs/tracing_opentelemetry/src/main/java/demo/jaxrs/tracing/ConsoleSpanExporter.java

```diff
@@ -0,0 +1,42 @@
+package demo.jaxrs.tracing;
+
+import java.util.Collection;
+
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ConsoleSpanExporter implements SpanExporter {
```

Comment:
May be we could use `LoggingSpanExporter` here?

Reply:
I did and it works, great suggestion!